README.md

```diff
@@ -4,4 +4,15 @@
 
 [![](https://images.microbadger.com/badges/image/casestack/node-service.svg)](http://microbadger.com/images/casestack/node-service "Microbadger")
 
-node6 base image that exposes port `40404`.
+node base image that exposes port `40404`.
+
+## Config
+Each object under `dockerfiles` has these keys:
+```
```

Comment:
Can you put these into a list instead of a code block? The code block doesn't wrap the text.